spec/specification_spec.rb

```diff
@@ -79,6 +79,26 @@ module Pod
         hash[spec_1].should == 'VALUE_2'
       end
 
+      it 'resets hash value when name changes' do
+        @spec.hash_value.should == 0
+        original_hash = @spec.hash
+        @spec.hash_value.should != 0
+        @spec.name = 'NewPodName'
+        @spec.hash_value.should == 0
+        new_hash = @spec.hash
+        original_hash.should != new_hash
+      end
+
+      it 'resets hash value when version changes' do
```

Comment:
This test fails while the one above passes. It seems that the way the version is stored via the `attributes_hash` doesn't have any effect...I am looking into more...